### ISSUE TITLE
QDQ propagation decrease model accuracy

### ISSUE THREAD

**Issue by tedder59** [NVIDIA/TensorRT#2419]
I used pytorch-quantization apis to perform quantization aware training with LaneATT model.
In training phase, f1 score of the model was 0.81.
But when I deployed this model with TensorRT, the f1 score was only 0.74.
When I look at the middle layer output, the result of TensorRT are different with the result of pytorch QAT.
Weather QDQ propagation induced this diff?

**Comment by zerollzeng** [comment]
Looks like a bug to me, Can you provide a reproduce for this?

**Comment by tedder59** [comment]
@zerollzeng Thank you for reply, I used some custom plugin in TensorRT engine, Finally I found those plugins induced the gap. After I fixed it, the f1 score of deploy code results also became 0.80. When I suspect TensorRT, the bugs are always in my own code. I think eval dataset can't publish, so I can't provide a way to reproduce. but I can use open dataset to apply the whole pipeline of QAT deploy later. Thank you Again.
